packages/office-ui-fabric-react/src/components/pickers/PickerItem.types.ts

```diff
@@ -12,4 +12,5 @@ export interface IPickerItemProps<T> extends React.AllHTMLAttributes<HTMLElement
   onItemChange?: (item: T, index: number) => void;
   key?: string | number;
   removeButtonAriaLabel?: string;
+  enableTagFocusInDisabledPicker?: boolean;
```

Comment:
Since this is the generic PickerItem, name this something more generic.

Reply:
I could name it something generic like `enableItemFocusInDisabledPicker`.
But this prop is only being used in `tagItem` and a generic name could be misleading.
Shall I update it then?

Reply:
If it's only used in tagItem, then it should be passed through the tag item props. If it's something that all picker items should have, then it should be added here. The picker is meant to be extensible. PickerItemProps is the base that all items have access to.